### DISCUSSION TITLE
服务端动态编辑

### DISCUSSION THREAD

**Discussion by poppyLuo** [Tencent/libpag#205] [Q&A] [answered]
请问支持动态编辑吗？比如说lottie可以在服务端替换其中的图片，修改lottie的json文件达到改文字、颜色等目标

**Comment by kevingpqi123** [comment, marked as answer]
PAG是跨平台的，支持服务端渲染，且各平台端能力保持一致性，你提到的动态编辑（文本、图片修改替换）更多的是业务层面的封装处理，比较典型的应用场景如一键出片、你在微信/手Q侧看到的王者战报,都是基于PAG实现的，文本、图片动态替换是其中很小的一个点
